packages/avatars/src/Avatar.example.md

```diff
@@ -26,82 +54,93 @@ For accessibility ensure that all children include an `alt` description.
   <Row alignItems="center" justifyContent="center">
     <Col md>
       <Avatar size="extrasmall">
-        <img src="images/jason.png" alt="Example Avatar" />
+        <img src="images/avatar-2.png" alt="Example Avatar" />
       </Avatar>
     </Col>
     <Col md>
       <Avatar size="small">
-        <img src="images/jason.png" alt="Example Avatar" />
+        <img src="images/avatar-2.png" alt="Example Avatar" />
       </Avatar>
     </Col>
     <Col md>
       <Avatar>
-        <img src="images/amir.png" alt="Example Avatar" />
+        <img src="images/avatar-2.png" alt="Example Avatar" />
       </Avatar>
     </Col>
     <Col md>
       <Avatar size="large">
-        <img src="images/jason.png" alt="Example Avatar" />
+        <img src="images/avatar-2.png" alt="Example Avatar" />
       </Avatar>
     </Col>
   </Row>
-</Grid>
-```
-
-### States
-
-```jsx
-<Grid>
-  <Row>
-    <Col md={4}>
-      <Avatar size="extrasmall">
-        <img src="images/jason.png" alt="Example Avatar" />
+  <Row alignItems="center" justifyContent="center">
+    <Col md>
+      <Avatar size="extrasmall" system>
+        <img src="images/system.png" alt="System Avatar" />
       </Avatar>
     </Col>
-    <Col md={4}>
-      <Avatar disabled size="extrasmall">
-        <img src="images/jason.png" alt="Example Avatar" />
+    <Col md>
+      <Avatar size="small" system>
+        <img src="images/system.png" alt="System Avatar" />
       </Avatar>
     </Col>
-    <Col md={4}>
-      <Avatar isBorderless size="extrasmall">
-        <img src="images/amir.png" alt="Example Avatar" />
+    <Col md>
+      <Avatar system>
+        <img src="images/system.png" alt="System Avatar" />
+      </Avatar>
+    </Col>
+    <Col md>
+      <Avatar size="large" system>
+        <img src="images/system.png" alt="System Avatar" />
       </Avatar>
     </Col>
   </Row>
+</Grid>
+```
+
+### States
+
+```jsx
+const { zdColorSecondaryAzure600 } = require('@zendeskgarden/css-variables');
+
+const StyledTextAvatar = styled(Avatar)`
+  background-color: ${zdColorSecondaryAzure600};
+`;
+
+<Grid>
   <Row>
     <Col md={4}>
-      <Avatar size="small">
-        <img src="images/jason.png" alt="Example Avatar" />
+      <Avatar status="active" badge="9+">
```

Comment:
Oops, all statuses throughout this example need to either be `away` or `available` – no `active`.